assemblyline_core/replay/loader/run_worker.py

```diff
@@ -34,6 +36,13 @@ def process_file(self, once=False):
                     elif file_path.endswith(".al_json"):
                         self.client.load_json(file_path)
 
+                    elif file_path.endswith(".al_json.cart"):
+                        cart_path = file_path
+                        file_path = file_path[:-5]
+                        unpack_file(cart_path, file_path)
```

Comment:
Should we check if file_path exists first, or use a temporary file?

Reply:
Well I expect that if the new file path doesn't exist, that means something happened at the `unpack_file()` call and that would raise an exception that would propagate up.